src_c/camera_windows.c

```diff
@@ -0,0 +1,840 @@
+/*
```

Comment:
Probably good to also add the license boilerplate.

Reply:
Done